slyd/slyd/jsonapi/registry.py

```diff
@@ -0,0 +1,16 @@
+from slyd.orm.exceptions import ImproperlyConfigured
+
+
+__all__ = [
+    'schema',
+]
+
+schemas = {}
```

Comment:
Can we change `schemas` to `serializers` to preserve the semantics of how this registry is used? (As we moved from calling `ObjectSchema` to `ObjectSerializer`)